ext/src/ruby_api/memory.rs

```diff
@@ -122,6 +127,22 @@ impl<'a> Memory<'a> {
             .map(RString::new)
     }
 
+    /// @yard
+    /// Read +size+ bytes starting at +offset+ into a `[Wasmtime::Memory::Slice`. This
```

Comment:
```suggestion
    /// Read +size+ bytes starting at +offset+ into a {Slice}. This
```

This will make YARD link to the `Slice` page, and also seems required for the `Slice` class to show up in the sidebar (likely due to bug/missing feature in yard rustdoc which I should fix!).